tooling.yml

```diff
@@ -0,0 +1,2 @@
+architecture-sniffer:
+    priority: 2
```

Comment:
Maybe add code sniffer too? Or does it default to level 1?

Reply:
It has default level